### ISSUE TITLE
Package status

### ISSUE THREAD

**Issue by alex-konovalov** [gap-packages/smallgrp#45]
What's the status of the package? Definitely not `dev` any more.
https://github.com/gap-packages/smallgrp/blob/e4168970dc22f5e15276ecaac2d93129346c69f6/PackageInfo.g#L98

**Comment by fingolfin** [comment]
Well, it never was formally reviewed as separate GAP package, I guess, though probably it was scrutinized when it was added to GAP originally...  am not sure if we have a Status value for this. We could mark it as "deposited"? Or ask the council if they have records about the status? Or make up something new?

**Comment by alex-konovalov** [comment]
My view is that it is "deposited" at least, because it **IS** redistributed with GAP. The question is whether it should be "accepted" - I think so, but this should be asked to the Council.

**Comment by fingolfin** [comment]
I changed it to `deposited`. Will you write the council to ask about the status?

**Comment by alex-konovalov** [comment]
Done.

**Comment by alex-konovalov** [comment]
The library was refereed and accepted in 2002: Information about the date and the editor could be derived from the following email: https://www.gap-system.org/ForumArchive/McDermot.1/John.1/Small_Gr.3/1.html

We need to check that PrimGrp and TransGrp status was also handled correctly when all three of these were extracted from the core GAP system in GAP 4.9.

**Comment by fingolfin** [comment]
Bettina corrected this in 58232e3acd9d6589543ed17d71352b3ecc8c7d25 so now it says this:
```
Status := "accepted",
CommunicatedBy := "Mike Newman (Canberra)",
AcceptDate := "02/2002",
```
